src/model/tree/DataVirtNodeProvider.ts

```diff
@@ -137,16 +137,14 @@ export class DataVirtNodeProvider implements vscode.TreeDataProvider<vscode.Tree
 	processDataVirtYAML(fullPath: string, name: string, yaml : DataVirtConfig): void {
 		if (yaml) {
 			try {
-				if (yaml.kind === constants.VDB_KIND) {
-					// found a DV config file -> put to tree
-					let title = name;
-					if (yaml.metadata.name) {
-						title = yaml.metadata.name;
-					}
-					const newItem = new DVProjectTreeNode(title, fullPath, yaml);
-					if (!this.doesNodeExist(this.treeNodes, newItem)) {
-						this.addChild(this.treeNodes, newItem, true);
-					}
+				// found a DV config file -> put to tree
+				let title = name;
+				if (yaml.metadata.name) {
+					title = yaml.metadata.name;
+				}
```

Comment:
for another PR: I think it is clearer to use an if else instead of changign the variable value.
```
let title;
if (yaml.metadata.name) {
    title = yaml.metadata.name;
} else {
    title = name;
}
```
and even extract a method:
```
const title = computeTitile(name, yaml);
```
```
computeTitle(name: string, yaml : DataVirtConfig) : string {
  if (yaml.metadata.name) {
   return yaml.metadata.name;
  } else {
    return name;
  }

} 
```

Reply:
I think both are valid approaches. The approach I used is using a default and only changes the default when a specific condition exists. It is easy to understand and shorter than the if/else or a separate function, which in this case would be overkill for such a small logic part.